### PR TITLE
1.0.7

### PR DESCRIPTION
**Pull request by dblanchard13**
- Split tutorial overlays into two parts.

- Add global exception handler that sends stack trace to rollbar.

- Update message text when starting a new scry from `Hey, Check this out` to `Tap the 🔮 to Scry ➡️`

- Add tap gesture recognizer on match's picture next to their tinder messages so that tapping it will open their tinder photos.

- Change `search` button on the add members keyboard to `done`

### CODE REVIEW

**Comment by jack829** [comment]
Yo in the future let's make each of these things their own commit since I suck at swift and that will make it easier for me to see what code is what

File: Scry/ScryChatsViewController.swift [hunk 1]
```diff
@@ -222,7 +222,8 @@ class ScryChatsViewController: ScryViewController {
         self.showTutorialOverLay()
         UserDefaults.standard.set(true, forKey: "isHomeTutorial")
       }
-    }else{
+    }
+    else{
```
Comment (jack829) [inline Scry/ScryChatsViewController.swift]: Is this the general style of else?

Comment (dblanchard13) [reply Scry/ScryChatsViewController.swift]: I'd prefer to break it onto the new line, yeah.

**Comment by dblanchard13** [comment]
Good point to make commits by feature.

File: Scry/ScryChatsViewController.swift (hunk 1 above)
Comment (jack829) [reply Scry/ScryChatsViewController.swift]: Looking through the code it looks like most places are just same line. I just wanna keep things like that consistent everywhere. 

Yeah I'm gonna be a stickler on code reviews

**Comment by Charliekenney23** [comment]
Looks good to me. But yeah, I agree, we need to enforce code style a bit more. The previous maintainers made it a bit sloppy.

**Pull request merged**